integration-tests/cli/category-exporter.it.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'mz/fs'
+import tmp from 'tmp'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'mz/child_process'
+import { version } from '@commercetools/category-exporter/package.json'
+import categories from './helpers/category-export.data'
+import { createData, clearData } from './helpers/utils'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'category-export-int-test'
+else projectKey = process.env.npm_config_projectkey
+
+describe('Category Exporter', () => {
+  let apiConfig
+  const bin = './integration-tests/node_modules/.bin/category-exporter'
+
+  beforeAll(async () => {
+    // Get test credentials
+    const credentials = await getCredentials(projectKey)
+    apiConfig = {
+      host: 'https://auth.sphere.io',
+      apiUrl: 'https://api.sphere.io',
+      projectKey,
+      credentials,
+    }
+
+    await clearData(apiConfig, 'categories')
```

Comment:
Isn't it enough to clear data only on `afterAll`?

Reply:
It is just a safety measure, to be sure that no data from the beginning.